corehq/apps/app_manager/dbaccessors.py

```diff
@@ -121,3 +122,14 @@ def get_built_app_ids(domain):
     app_ids = [data.get('value', {}).get('copy_of') for data in result]
     app_ids = list(set(app_ids))
     return [app_id for app_id in app_ids if app_id]
+
+
+def get_exports_by_application(domain):
+    from .models import Application
+    return Application.get_db().view(
+        'exports_forms/by_xmlns',
+        startkey=['^Application', domain],
+        endkey=['^Application', domain, {}],
+        reduce=False,
+        stale=settings.COUCH_STALE_QUERY,
```

Comment:
is that documented somewhere? afaik we don't really have a single point of truth for documentation and best practices.